base/xx-verify

```diff
@@ -85,7 +85,7 @@ for f in "$@"; do
 
   expOS=""
   case "$TARGETOS" in
-    "linux")
+    "linux" | "freebsd")
```

Comment:
Shouldn't this also check "FreeBSD-style" (or at least FreeBSD)?

Reply:
Yep, good call. https://github.com/tonistiigi/xx/pull/47/files